op-deployer/pkg/deployer/integration_test/apply_test.go

```diff
@@ -54,82 +59,7 @@ func (d *deployerKey) String() string {
 	return "deployer-key"
 }
 
-func TestLiveChain(t *testing.T) {
-	t.Skip("requires backport")
-
-	op_e2e.InitParallel(t)
-
-	for _, network := range []string{"mainnet", "sepolia"} {
-		t.Run(network, func(t *testing.T) {
-			testLiveChainNetwork(t, network)
-		})
-	}
-}
-
-func testLiveChainNetwork(t *testing.T, network string) {
-	op_e2e.InitParallel(t)
-	lgr := testlog.Logger(t, slog.LevelInfo)
-	rpcURL := os.Getenv(fmt.Sprintf("%s_RPC_URL", strings.ToUpper(network)))
-	require.NotEmpty(t, rpcURL)
-
-	forkedL1, cleanup, err := devnet.NewForked(lgr, rpcURL)
-	require.NoError(t, err)
-	t.Cleanup(func() {
-		require.NoError(t, cleanup())
-	})
-
-	l1Client, err := ethclient.Dial(forkedL1.RPCUrl())
-	require.NoError(t, err)
-
-	ctx, cancel := context.WithCancel(context.Background())
-	defer cancel()
-
-	l1ChainID, err := l1Client.ChainID(ctx)
-	require.NoError(t, err)
-
-	pk, err := crypto.HexToECDSA("ac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80")
-	require.NoError(t, err)
-	dk, err := devkeys.NewMnemonicDevKeys(devkeys.TestMnemonic)
-	require.NoError(t, err)
-
-	testCacheDir := testutils.IsolatedTestDirWithAutoCleanup(t)
-
-	intent, st := newIntent(
-		t,
-		l1ChainID,
-		dk,
-		uint256.NewInt(9999),
-		artifacts.DefaultL1ContractsLocator,
-		artifacts.DefaultL2ContractsLocator,
-	)
-	cg := ethClientCodeGetter(ctx, l1Client)
-
-	require.NoError(t, deployer.ApplyPipeline(
-		ctx,
-		deployer.ApplyPipelineOpts{
-			DeploymentTarget:   deployer.DeploymentTargetLive,
-			L1RPCUrl:           forkedL1.RPCUrl(),
-			DeployerPrivateKey: pk,
-			Intent:             intent,
-			State:              st,
-			Logger:             lgr,
-			StateWriter:        pipeline.NoopStateWriter(),
-			CacheDir:           testCacheDir,
-		},
-	))
-
-	validateSuperchainDeployment(t, st, cg, false)
-	validateOPChainDeployment(t, cg, st, intent, false)
-}
-
-func TestEndToEndApply(t *testing.T) {
-	op_e2e.InitParallel(t)
-
-	lgr := testlog.Logger(t, slog.LevelDebug)
-
-	ctx, cancel := context.WithCancel(context.Background())
-	defer cancel()
-
+func anvilRPC(t *testing.T, lgr log.Logger) (string, *ethclient.Client) {
```

Comment:
Could you move this and the `cannedPrivKey` functions to `op-service/testutils`? I can use them for the gnosis client integration tests as well